upload_to_gel/deliver_data_to_gel.py

```diff
@@ -253,26 +253,33 @@ def check_delivery_data(self):
             self.delivery_id,
             'upload_state', 'md5_state', 'md5_confirm_date', 'qc_state', 'qc_confirm_date'
         )
+        msg = ''
+        emit = self.error
```

Comment:
That seems unnecessary to replace `self.error` with another variable name

Reply:
Fixed as per review below